pyanaconda/modules/payloads/payload/dnf/dnf.py

```diff
@@ -225,6 +236,79 @@ def _set_up_kickstart_packages(self, data):
         convert_packages_selection_to_ksdata(self.packages_selection, data)
         convert_packages_configuration_to_ksdata(self.packages_configuration, data)
 
+    def get_available_repositories(self):
+        """Get a list of available repositories.
+
+        :return: a list with names of available repositories
+        """
+        return self.dnf_manager.repositories
+
+    def get_enabled_repositories(self):
+        """Get a list of enabled repositories.
+
+        :return: a list with names of enabled repositories
+        """
+        return self.dnf_manager.enabled_repositories
+
+    def get_default_environment(self):
+        """Get a default environment.
+
+        :return: an identifier of an environment or an empty string
+        """
+        return self.dnf_manager.default_environment or ""
+
+    def get_environments(self):
+        """Get a list of environments defined in comps.xml files.
+
+        :return: a list with identifiers of environments
+        """
+        return self.dnf_manager.environments
+
+    def resolve_environment(self, environment_spec):
+        """Translate the given specification to an environment identifier.
+
+        :param environment_spec: an environment specification
+        :return: an identifier of an environment or an empty string
+        """
+        return self.dnf_manager.resolve_environment(environment_spec) or ""
+
+    def get_environment_data(self, environment_spec):
+        """Get data about the specified environment.
+
+        :param environment_spec: an environment specification
+        :return CompsEnvironmentData: the related environment data
+        :raise UnknownCompsEnvironmentError: if the environment is unknown
+        """
+        return self.dnf_manager.get_environment_data(environment_spec)
+
+    def resolve_group(self, group_spec):
+        """Translate the given specification into a group identifier.
+
+        :param group_spec: a group specification
+        :return: an identifier of a group or an empty string
+        """
+        return self.dnf_manager.resolve_group(group_spec) or ""
+
+    def get_group_data(self, group_spec):
+        """Get data about the specified group.
+
+        :param group_spec: a specification of a group
+        :return CompsGroupData: the related group data
+        :raise: UnknownCompsGroupError if the group is unknown
+        """
+        return self.dnf_manager.get_group_data(group_spec)
+
+    def verify_repomd_hashes(self):
+        """Verify a hash of the repomd.xml file for each enabled repository.
+
+        This method tests if URL links from active repositories can be reached.
+        It is useful when network settings is changed so that we can verify if
```

Comment:
I think there is a typo -> "network settings are changed" ?

Reply:
Fixed.